### PR TITLE
requests and limits for all

### PR DESCRIPTION
**Pull request by jessegoodier**
Thanks to Sean for most of this.

### CODE REVIEW

**Review by srpomeroy — APPROVED** [review]
Thanks for formalizing it!

File: helm-limits/requests-and-limits.yaml [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+     # limits:
+     #  cpu: 200m
+     #  memory: 55Mi
+
+kubecostModel:
+  resources:
+    requests:
+      cpu: 200m
+      memory: 55Mi
+  # limits:
```
Comment (linhlam-kc) [inline helm-limits/requests-and-limits.yaml:28]: Should we add a higher example limit here and comment on what typical limit for big cluster should look like? e.g: Clusters with x pods would need yy CPU and zz Memory.

**Review by linhlam-kc — APPROVED** [review]
LGTM but I would recommend to add more comments with example of how the resource limit for big size cluster should look like for core components like cost-model, prometheus server, thanos sidecar.

File: helm-limits/requests-and-limits.yaml (hunk 1 above)
Comment (jessegoodier) [reply helm-limits/requests-and-limits.yaml:28]: yes! 

I'll merge, feel free to make it better

**Pull request merged**